server/src/main/java/com/cloud/configuration/Config.java

```diff
@@ -1836,9 +1836,9 @@
             Boolean.class,
             "execute.in.sequence.network.element.commands",
             "false",
-            "If set to true, DhcpEntryCommand, SavePasswordCommand, UserDataCommand, VmDataCommand will be synchronized on the agent side."
```

Comment:
Also found some reference on the `schema-410to420.sql` file

Reply:
Good catch. I will fix that, and create a script to update the database.